tests/host_tools/fcmetrics.py

```diff
@@ -463,14 +464,15 @@ def flatten_dict(node, prefix: str):
             continue
         metrics.put_metric(key, value, get_emf_unit_for_fc_metrics(key))
 
-    assert not {
+    metrics.flush()
+
+    failure_metrics = {
         key: value
         for key, value in flattened_metrics.items()
         if "err" in key or "fail" in key or "panic" in key or "num_faults" in key
         if value
     }
-
-    metrics.flush()
+    assert not failure_metrics, json.dumps(failure_metrics)
```

Comment:
would it be worth to pretty print with `indent=True`?

Reply:
Also why is it not legible? Shouldn't pytest pretty print it for us?

Reply:
The output currently looks like this: https://buildkite.com/firecracker/performance-nightly-metrics/builds/273#019143e5-33d0-4707-9987-f6ff987f44c9/50-239. I can do the `ident=True`  change